lib/cube_browser/__init__.py

```diff
@@ -93,3 +96,47 @@ def slider_coords(self):
                 available.append(self.cube.dim_coords[dim])
         return available
 
+
+class Browser(object):
+    """
+    Compiler for cube_browser plots and associated sliders.
+
+    Compiles a single cube_browser.Plot instance or list of instances into a
+    vertical arrangement of axes with shared coordinate sliders, to be
+    displayed in a Jupyter notebook.
+    """
+    def __init__(self, plot):
+        """
+        Compiles non-axis coordinates into sliders, the values from which are
+        used to reconstruct plots upon movement of slider.
+
+        Args:
+
+        * plot: cube_browser.Plot instance to display with slider
+        """
+        self.plot = plot
+        self._sliders = {}
+        self._slidermap = {}
+        for coord in plot.slider_coords():
+            if coord not in self._slidermap:
+                self._slidermap[coord.name()] = coord.shape[0]
```

Comment:
@bjlittle I remember what this was for now.  This is so that when we have multiple plots it checks for existing sliders to see if the coordinates match (in which case they share it), so removing it may come back to bite us later but for now it works fine without it.